xinference/model/llm/core.py

```diff
@@ -18,12 +18,23 @@
 from abc import abstractmethod
 from typing import TYPE_CHECKING
 
+# Default context length for user-defined models
+DEFAULT_CONTEXT_LENGTH = 2048
+
 if TYPE_CHECKING:
     from .llm_family import LLMFamilyV1, LLMSpecV1
 
 logger = logging.getLogger(__name__)
 
 
+def sanitize_model_family(model_family: "LLMFamilyV1") -> "LLMFamilyV1":
```

Comment:
How about just update the definition of LLMFamilyV1 and set a default value for context length?

Here's an example:
```python
class User(BaseModel):
    id: Optional[str] = 0
```